js/classes/entities/midground/block.js

```diff
@@ -97,6 +97,51 @@ class Block {
     static get PLANK_SPRUCE_STAIRS_RIGHT() {
         return 20;
     };
+    static get SNOW_SURFACE() {
+        return 21;
+    };
+    static get BR_LEFT_FULL() {
+        return 22;
+    };
+    static get BR_RIGHT_FULL() {
+        return 23;
+    };
+    static get BRANCH_LEFT_HALF() {
+        return 24;
+    };
+    static get BRANCH_RIGHT_HALF() {
+        return 25;
+    };
+    static get BRANCH_TOP() {
+        return 26;
+    };
+    static get BRANCH_FILL_LOG() {
+        return 27;
+    };
+    static get BRANCH_FILL() {
+        return 28;
+    };
+    static get TWIG_LEFT() {
+        return 29;
+    }
+    static get TWIG_RIGHT() {
+        return 30;
+    }
+    static get BR_LEFT_END() {
+        return 31;
+    }
+    static get BR_RIGHT_END() {
+        return 32;
+    }
+    static get BR_LEFT() {
+        return 33;
+    }
+    static get BR_RIGHT() {
+        return 34;
+    }
+    static get HIDDEN_BLOCK_CLOUD() {
+        return 35;
+    }
```

Comment:
What do all of these constants mean? JSDoc would be helpful.

Reply:
My bad! I'll update that. Hidden block is for putting in places that don't normally have blocks. Like hiding a bounding box to step on behind a cloud for example.